### DISCUSSION TITLE
Parceria com o desenvolvedor do projeto principal?

### DISCUSSION THREAD

**Discussion by Bardock88** [rafaelvcaetano/melonDS-android#1015] [General]
Que tal se @arisotura e você @rafaelvcaetano fizessem uma colaboração/parceria em prol de melhorar ainda mais as versões tanto de PC e Android, assim como existe com outros emuladores de consoles, como por exemplo: AetherSX2 e PCSX2 de PS2, estão fazendo, assim tanto vocês e o público estariam se beneficiando e ajudando uns aos outros em algum problema que pode haver no desenvolvimento de ambas versões dos S.O.'s de destino.

**Comment by goliv04053** [comment]
*a @arisotura

**Reply by Bardock88** [reply]
> *a @Arisotura

Agora corrigir 👍.

**Reply by rafaelvcaetano** [reply]
No caso do melonDS, haver uma parceria para melhorar as duas versões nºao faz sentido porque só existe mesmo um emulador. Arisotura e a equipa da versão de PC é que são os principais responsáveis por desenvolver o emulador. Eu apenas faço as alterações necessárias para ele correr em Android e tento fazê-la da maneira mais fácil para o utilizador. Em relação a um commitment mais profundo da minha parte, tal não é possível porque não tenho tempo suficiente para me comprometer a algo em concreto. No entanto, caso eu descubra alguma coisa que beneficie o emulador no geral, obviamente que tratarei de incluir essa melhoria no emulador para que todas as plataformas beneficiem.

**Reply by Bardock88** [reply]
Ah tudo bem, entendo sua condição atual com estudos da faculdade e a falta de tempo disponível para se doar ao projeto de _port_ e me desculpe se de alguma forma acabei sendo indiscreto com a pergunta.
Muito obrigado pela sua resposta e que você consiga realizar todos os seus objetivos presentes e futuros 👍.
